config/converter.yml.example

```diff
@@ -1,20 +1,31 @@
-development:
-  :url:
+shared:
```

Comment:
My list of extensions:
    - '.xml'
    - '.xrd'
    - '.pdf'
    - '.xrdml'
    - '.uxd'
    - '.json'
    - '.tif'
    - '.tiff'
    - '.tar'
    - '.gz'
    - '.cif'
    - '.cfx_LANA'
    - '.xy'
    - '.csv'
    - '.xls'
    - '.xlsx'
    - '.txt'
    - '.brml'
    - '.dta'
    - '.pssession'
    - '.log'
    - '.mpr'
    - '.mpl'
    - '.mps'
    - '.aif'
Please check if changes are necessary?